test/config-luatest/gh_10304_allow_specific_function_access_test.lua

```diff
@@ -0,0 +1,70 @@
+local fun = require('fun')
+local yaml = require('yaml')
+local t = require('luatest')
+local treegen = require('test.treegen')
+local server = require('test.luatest_helpers.server')
+local helpers = require('test.config-luatest.helpers')
+
+local g = helpers.group()
+
+g.test_lua_call_func = function(g)
+    local dir = treegen.prepare_directory(g, {}, {})
+    local config = {
+        credentials = {
+            users = {
+                guest = {
+                    roles = {'super'},
+                },
+                alice = {
+                    password = "ALICE",
+                    privileges = {
+                        {
+                            permissions = {"execute"},
+                            lua_call = {"foo", "bar"}
+                        },
+                    }
+                },
```

Comment:
It seems desirable to have a test case that verifies that the same works for a role.

Reply:
test with role added